### PR TITLE
Fixes #33832 - update VCRs and bindings gems for Pulpcore 3.16

### PR DESCRIPTION
**Pull request by ianballou**
### What are the changes introduced in this pull request?

Updated VCRs and bindings gems for Pulpcore 3.16

### What is the thinking behind these changes?

I edited one of the tests to stop running its setup because all of its tests are skipped.

### What are the testing steps for this pull request?

Install a Pulpcore 3.16 sandbox and try out content actions: https://projects.theforeman.org/projects/katello/wiki/Pulp_3_Integration#Using-a-2nd-server-as-a-pulp-server-for-use-in-development

Try actions, such as:
1) Smart Proxy syncing
2) Content view publishing:
  - With and without filters
  - With and without dep solving
3) File uploading

### CODE REVIEW

**Comment by ianballou** [comment]
Blocked on Pulpcore 3.16 packaging.

**Comment by ianballou** [comment]
Related: https://github.com/theforeman/foreman-packaging/pull/7221

**Review by jlsherrill — APPROVED** [review]
ack from me once the 3.16 repos are available

**Comment by ianballou** [comment]
I'll re-record the VCRs just before merging when the repos are ready.

**Comment by jlsherrill** [comment]
[test katello]

File: test/services/katello/pulp3/erratum_test.rb
Comment (jlsherrill) [inline test/services/katello/pulp3/erratum_test.rb:3]: ruhroh!

Comment (ianballou) [reply test/services/katello/pulp3/erratum_test.rb:3]: ah darn, that snuck in. Thanks.

**Review by jlsherrill — APPROVED** [review]

**Pull request merged**
